CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to Torchaudio
+We want to make contributing to this project as easy and transparent as possible.
+
+## TL;DR
+
+Please let us know if you encounter a bug by filing an [issue](https://github.com/pytorch/audio/issues).
+
+We appreciate all contributions. If you are planning to contribute back
+bug-fixes, please do so without any further discussion.
+
+If you plan to contribute new features, utility functions or extensions to the
+core, please first open an issue and discuss the feature with us. Sending a PR
+without discussion might end up resulting in a rejected PR, because we might be
+taking the core in a different direction than you might be aware of.
+
+Fixing bugs and implementing new features are not the only way you can
+contribute. It also helps the project when you report problems you're facing,
+and when you give a :+1: on issues that others reported and that are relevant
+to you.
+
+You can also help by improving the documentation. This is no less important
+than improving the library itself! If you find a typo in the documentation,
+do not hesitate to submit a pull request.
+
+If you're not sure what you want to work on, you can pick an issue from the
+[list of open issues labelled as "help
+wanted"](https://github.com/pytorch/audio/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).
+Comment on the issue that you want to work on it and send a PR with your fix
+(see below).
+
+## Development installation
+
+We recommend using a `conda` environment to contribute efficiently to
+torchaudio.
+
+### Install PyTorch Nightly 
+
+```bash
+conda install pytorch -c pytorch-nightly
+```
+
+### Install Torchaudio
+
+```bash
+conda install cmake
+```
+
+```bash
+git clone https://github.com/pytorch/audio.git
+cd audio
+BUILD_SOX=1 python setup.py develop
+# or, for OSX
+# BUILD_SOX=1 MACOSX_DEPLOYMENT_TARGET=10.9 CC=clang CXX=clang++ python setup.py develop
+# for C++ debugging, please use DEBUG=1
+# DEBUG=1 python setup.py install
+```
+
+Note: you don't need to use `BUILD_SOX=1` if you have `libsox-dev` installed
+already.
+
```

Comment:
Can you also add note that when testing `libsox`-related features, making the `sox` command coming from the torchaudio build process (which is found in `third_party/install/bin` after build completes)

```
Set `PATH` variable so that the sox command, created as a part of torchaudio build process, is used during the test.
export PATH="<path_to_torchaudio>/third_party/install/bin:${PATH}"
```

Reply:
Also 

`export PATH="${PATH}:<path_to_kaldi>/src/featbin/"`

for running Kaldi-compatibility tests